ZuluContent/Items/Skill Items/Fishing/Misc/SpecialFishingNet.cs

```diff
@@ -292,146 +256,286 @@ protected void Spawn(Point3D p, Map map, BaseCreature spawn)
+                        return new BoneGloves();
+                    }
 
-            return water;
-        }
+                    return new BoneGloves();
+                }
+                case 2:
+                {
+                    if (skill > 110)
+                    {
+                        if (Utility.Random(4) == 2)
+                            return new TerrorLegs();
 
-        private static int[] m_UndeepWaterTiles = new[]
-        {
-            0x1797, 0x179C
-        };
+                        return new BoneLegs();
+                    }
 
-        private static bool ValidateUndeepWater(Map map, object obj, ref int z)
-        {
-            if (!(obj is StaticTarget))
-                return false;
+                    return new BoneLegs();
+                }
+                case 3:
+                case 4:
+                {
+                    if (skill > 120)
+                    {
+                        if (Utility.Random(5) == 2)
+                            return new TerrorChest();
 
-            StaticTarget target = (StaticTarget) obj;
+                        return new BoneChest();
+                    }
 
-            if (BaseHouse.FindHouseAt(target.Location, map, 0) != null)
-                return false;
+                    return new BoneChest();
+                }
+                case 5:
+                {
+                    if (skill > 110)
+                    {
+                        if (Utility.Random(5) == 2)
+                            return new TerrorHelm();
 
-            int itemID = target.ItemID;
+                        return new BoneHelm();
+                    }
 
-            for (int i = 0; i < m_UndeepWaterTiles.Length; i += 2)
-            {
-                if (itemID >= m_UndeepWaterTiles[i] && itemID <= m_UndeepWaterTiles[i + 1])
+                    return new BoneHelm();
+                }
+                case 6:
                 {
-                    z = target.Z;
-                    return true;
+                    if (skill > 120)
+                    {
+                        if (Utility.Random(5) == 2)
+                            return new DartThrowerOfAmroth();
+
+                        return new HeavyCrossbow();
+                    }
+
+                    return new HeavyCrossbow();
                 }
+                default:
+                    return new BoneGloves();
             }
-
-            return false;
         }
-    }
 
-    public class FabledFishingNet : SpecialFishingNet
-    {
-        public override int LabelNumber
+        private static BaseMagicFish? GetFish(Mobile from)
         {
-            get { return 1063451; }
-        } // a fabled fishing net
+            var skill = from.Skills[SkillName.Fishing].Value;
+            var chance = Utility.RandomMinMax(1, (int) skill);
 
+            if (chance >= 100)
+            {
+                return Utility.Random(4) switch
```

Comment:
On ZH3 there are 8 magic fish - wasn't sure if we wanted to use all of them?

Reply:
They're pretty useless to be honest haha, up to you if you want to implement them.

Reply:
Will leave them out for now